runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/CaptainBarnaby.java

```diff
@@ -0,0 +1,21 @@
+package net.runelite.client.plugins.menuentryswapper;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum CaptainBarnaby
```

Comment:
This should probably be named something like `CaptainBarnabyMode`, to be more descriptive